packages/ui/src/working-groups/components/OpeningsList/OpeningsList.tsx

```diff
@@ -6,14 +6,14 @@ import { UpcomingWorkingGroupOpening, WorkingGroupOpening } from '@/working-grou
 import { OpeningsListRow } from './OpeningsListRow'
 
 export interface OpeningsListProps {
-  openings: WorkingGroupOpening[] | UpcomingWorkingGroupOpening[]
+  openings?: WorkingGroupOpening[] | UpcomingWorkingGroupOpening[]
```

Comment:
Could you double check if this is needed? If used together with `isLoading` check you can add guard for this and always require openings list (like in the LoadingOpenings component)

Reply:
Turns out I took care of that in the hook some commits later. Not needed indeed